src/components/atoms/Loading/index.tsx

```diff
@@ -1,27 +1,70 @@
 "use client";
 
 import { motion } from "framer-motion";
+import { StaticImport } from "next/dist/shared/lib/get-img-props";
 import Image from "next/image";
 import React from "react";
-import LogoWhite from "@/images/easytolive/logo/logobranca-fundoprimary.svg";
+import cx from "classnames";
+
+interface ILoadingComponent {
+  fullSize?: boolean;
+  bgColor?: "main" | "secondary" | "none";
+  Icon?: StaticImport;
+}
+
+const LoadingComponent: React.FC<ILoadingComponent> = ({
+  fullSize = true,
+  bgColor = "main",
+  Icon,
+}) => {
+  const backGroundColor = () => {
+    switch (bgColor) {
+      case "main":
+        return "bg-primary-main";
+      case "secondary":
+        return "bg-secondary-main";
+      case "none":
+        return "bg-none";
+      default:
+        return "main";
+    }
+  };
+  const fullSizeLoadingComponent = () => {
+    switch (fullSize) {
+      case true:
+        return "min-h-[93vh]";
+      case false:
+        return "";
+    }
+  };
 
-const LoadingComponent = () => {
   return (
-    <div className="w-full min-h-[93vh] bg-primary-main flex flex-col items-center justify-around">
+    <div
+      className={cx(
+        backGroundColor(),
+        fullSizeLoadingComponent(),
```

Comment:
```suggestion
        fullSize ? 'min-h-[93vh]' : '',
```

We don't need this function

Reply:
Check https://www.npmjs.com/package/classnames